wasm2c/examples/rot13/main.c

```diff
@@ -20,62 +20,46 @@
 #include "rot13.h"
 
 /* Define structure to hold the imports */
-struct Z_host_instance_t {
+typedef struct w2c_host {
   wasm_rt_memory_t memory;
   char* input;
-};
+} w2c_host;
```

Comment:
Is adding this extra typedef part of the this change too?   Or was this file just out-of-date?

Reply:
I think it probably should have been a typedef all along (so main.c can refer to it as `w2c_host` instead of `struct w2c_host`, and to match the wasm2c generated code which always typedef's the name of the struct), but it's not critical...

Reply:
Any chance we can split that out as well? .. sorry for being so pedantic, but it helps my rather slow brain understand PR much better when its focused on just one thing.   Feel free to push back on this if you think its too much.

Reply:
Sure, no problem. Stand by...

Reply:
Done in a5a81648e39ae961a7173f00f34be52d0f5f97e4. I'm guessing there's going to be more in c-writer.cc that you'll want split out, which I may try to get to first to save you the trouble...

Reply:
(done for now)